demo/agent_monitor/tenant_agent_monitor.py

```diff
@@ -33,7 +33,7 @@
 
 logger = keylime_logging.init_logging("agent_monitor")
 
-initscript = None
+init_script = None  # pylint: disable-msg=C0103
```

Comment:
This is treated as a global variable, so it can be in caps

Reply:
Done.